allergy intolerances/README.md

```diff
@@ -133,7 +133,7 @@ An Allergy Intolerance is mapped to a CompoundStatement with inner ObservationSt
 | ObservationStatement / availabilityTime                                      | `AllergyIntolerance.assertedDate`                                                                            |
 | ObservationStatement / pertinentInformation / pertinentAnnotation / text     | `AllergyIntolerance.note\[0].text`                                                                           |
 | ObservationStatement / Participant \[@typeCode=AUT] / agentRef / id \[@root] | `AllergyIntolerance.recorder`                                                                                |
-| ObservationStatement / Participant \[@typeCode=PRF] / agentRef / id \[@root] | `AllergyIntolerance.asserter` or `AllergyIntolerance.recorder`                                               |
+| ObservationStatement / Author \[@typeCode=AUT] / agentRef / id \[@root]      | `AllergyIntolerance.asserter` -> value from Author field otherwise take it from Participant field            |
```

Comment:
You've changed the section under the title
> ## JSON FHIR > XML HL7

Whereas we've changed the behaviour of the XML -> JSON mapping.

Reply:
good catch, I've corrected it now